### PR TITLE
Load settings from an existing validation

### PR DESCRIPTION
**Pull request by s-scherrer**
This adds a button to the results/published validations page that allows to rerun a validation.

### CODE REVIEW

**Comment by sheenaze** [comment]
Generally it works and it does what it should. There are just a few issues or inconsistencies that should be improved:
1. The button description should be: "Load settings of the validation" or something similar, because "rerun validation" suggests two things: a) after clicking the button a new validation will start, b) rerunning validation makes sense because for instance it will give different results (although maybe this one is not that obvious).
2. When uploading settings with more than one dataset to be validated there is an issue with addition and removal buttons; Uploading 5 datasets does not cause vanishing the "add dataset" button and uploading settings with e.g. 3 datasets do not show the removal button. I noticed and adjusted this issue when I was working on existing validations task, here: https://github.com/awst-austria/qa4sm/pull/119/files#diff-aa042bc108ccf1ae6bb2f5e2920c8fc7cfa2b7266f7975176a6baa1b3bf9345aR50-R51 and here: https://github.com/awst-austria/qa4sm/pull/119/files#diff-aa042bc108ccf1ae6bb2f5e2920c8fc7cfa2b7266f7975176a6baa1b3bf9345aR1069-R1071  
so you can use my solution, or we can wait with your task until we apply mine. It was not an issue before, because we never had a situation that we rendered at the same time more than one dataset on the left side :)
3. Uploading the same dataset introduced not only date but also time into Validation Period fields. It doesn't cause problems, but it is inconsistent with the settings that are originally introduced.
4. Some comments about the appearance:
- I would add the button as the last one, because it's a new button and it is not really connected to the others so there is no particular reason why it should be somewhere in the middle (correct me if I'm wrong);
- I would think of a different button layout - e.g. two rows 4 and 3 buttons in each row, because now the validation container is quite wide and it just takes a lot of space while not looking good;
5. I would add a button also to a result page, to so that it is possible to upload settings also from the particular results view.

If I find something more, I'll let you know. Generally, good job :)

**Comment by s-scherrer** [comment]
Thanks for the comments. Regarding 2., I would suggest that we wait until yours is merged (if that is not too late).

**Comment by sheenaze** [comment]
Ok, we can wait :)

**Review by bakcsa83 — APPROVED** [review]

**Pull request closed without merging**
